lntest/harness_assertion.go

```diff
@@ -667,6 +668,27 @@ func (h *HarnessTest) AssertStreamChannelForceClosed(hn *node.HarnessNode,
 	closingTxid := h.WaitForChannelCloseEvent(stream)
 	h.Miner.AssertTxInBlock(block, closingTxid)
 
+	// This makes sure that in case of a neutrino backend we do not have any
+	// lingering unconfirmed anchor cpfp transactions blocking some of our
+	// utxos.
+	if anchors && h.IsNeutrinoBackend() {
```

Comment:
Agree good idea. I was thinking of performance issues here but I tested locally, because there are no unconfirmed utxos we lose almost no time doing this check for other backends.